AnacondaRecipes/nb_conda_kernels-feedstock
# nb_conda_kernels 2.5.2

**Destination channel:** main

### Links

- [PKG-6025](https://anaconda.atlassian.net/browse/PKG-6025) 
- [Upstream repository](https://github.com/anaconda/nb_conda_kernels/tree/2.5.2)
- Requirements: 
  - https://github.com/anaconda/nb_conda_kernels/blob/2.5.2/setup.py
  - https://github.com/anaconda/nb_conda_kernels/blob/2.5.2/requirements.txt
  - https://github.com/anaconda/nb_conda_kernels/blob/2.5.2/conda-recipe/meta.yaml

### Explanation of changes:

- Remove `abs.yaml`
- Skip `s390x` as `notebook <7` isn't available
- Add missing `jupyter_server` and `traitlets` to `run`
- Add `imports` to `test`
- Do not constraint `notebook` in `tests/requires` because `nb_conda_kernels` picks up `jupyter_server` if `notebook >=7` in the environment
- Add more test commands based on the conda-forge recipe, see https://github.com/conda-forge/nb_conda_kernels-feedstock/blob/main/recipe/meta.yaml#L34-L37
- Add the `pytest` command but skip broken tests (`test_configuration` and `test_install_kernelspec`). I've opened an issue about `test_install_kernelspec` https://github.com/anaconda/nb_conda_kernels/issues/274 
- Update `summary` and `description`
- Remove `skip-lints` as we build an arch-specific package and have imports.

### Notes:

- It was requested to build **v2.5.0**.

[PKG-6025]: https://anaconda.atlassian.net/browse/PKG-6025?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
